shared/teams/channel/header.tsx

```diff
@@ -0,0 +1,198 @@
+import * as React from 'react'
+import * as Kb from '../../common-adapters'
+import * as Styles from '../../styles'
+import * as Container from '../../util/container'
+import * as Constants from '../../constants/teams'
+import * as Chat2Gen from '../../actions/chat2-gen'
+import {ConversationIDKey} from '../../constants/types/chat2'
+import {TeamID} from '../../constants/types/teams'
+import {pluralize} from '../../util/string'
+import {Activity} from '../common'
+
+type HeaderTitleProps = Kb.PropsWithOverlay<{
```

Comment:
Instead of doing the overlay with a HOC, you should use `Kb.usePopup`.

Reply:
Prefer to leave it out until the menu is added